FBSnapshotTestCaseDemo/FBSnapshotTestCaseDemoTests/FBSnapshotTestCaseDemoTests.m

```diff
@@ -26,10 +25,39 @@ - (void)setUp
   self.recordMode = NO;
 }
 
-- (void)testExample
+- (void)testViewSnapshot
 {
-  FBExampleView *v = [[FBExampleView alloc] initWithFrame:CGRectMake(0, 0, 64, 64)];
-  FBSnapshotVerifyView(v, nil);
+  UIView *redView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 40, 40)];
+  redView.backgroundColor = [UIColor redColor];
+  redView.frame = CGRectMake(0, 0, 40, 40);
+  FBSnapshotVerifyView(redView, nil);
+  FBSnapshotVerifyLayer(redView.layer, nil);
+}
+
+- (void)testViewSnapshotWithVisualEffects
+{
+  if ([UIVisualEffect class]) {
+    UIVisualEffect *blurEffect = [UIBlurEffect effectWithStyle:UIBlurEffectStyleLight];
+    UIVisualEffectView *visualEffectView = [[UIVisualEffectView alloc] initWithEffect:blurEffect];
+    UIView *bluredRedView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 40, 40)];
+    bluredRedView.backgroundColor = [UIColor redColor];
+    visualEffectView.frame = bluredRedView.bounds;
+    [bluredRedView addSubview:visualEffectView];
+    
+    self.usesDrawViewHierarchyInRect = YES;
+    FBSnapshotVerifyView(bluredRedView, nil);
```

Comment:
You won't see on snapshot blur/vibrant effect if both view will have same frames.
I would recommend to change to:
UIView *redView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 20, 40)];
redView.backgroundColor = [UIColor redColor];
visualEffectView.frame = CGRectMake(0, 0, 40, 20);

UIView *parentView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, 40, 40)];
[parentView addSubview:redView];
[parentView addSubview:visualEffectView];